### PR TITLE
Small cleanups to the bluetooth proxies page

### PR DESCRIPTION
**Pull request by bdraco**
- Make the advanced links more discoverable for
  non-chrome users

- Fix the mouser link for the Olimex board

- Switch to linking to the Olimex board with
  an external antennae

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -199,13 +202,13 @@ <h3>Olimex ESP32 Power-over-Ethernet ISO</h3>
...
             <a
-              href="https://www.olimex.com/Products/IoT/ESP32/ESP32-POE-ISO/open-source-hardware"
+              href="https://www.olimex.com/Products/IoT/ESP32/ESP32-POE-ISO-EA/open-source-hardware"
               >Olimex</a
             >
           </li>
           <li>
             <a
-              href="https://www.mouser.com/ProductDetail/Olimex-Ltd/ESP32-POE-ISO"
+              href="https://www.mouser.com/ProductDetail/Olimex-Ltd/ESP32-POE-ISO-EA?qs=%252B6g0mu59x7KfK%2FIl1ERS8Q%3D%3D"
```
Comment (balloob) [inline static/index.html:211]: Do we need the qs part?

File: static/index.html [hunk 2]
```diff
@@ -238,20 +241,20 @@ <h3>Wireless-Tag WT32-ETH01</h3>
...
-            href="https://my.home-assistant.io/redirect/supervisor_addon/?addon=5c53de3b_esphome&repository_url=https%3A%2F%2Fgithub.com%2Fesphome%2Fhome-assistant-addon"
-            >the ESPHome dashboard</a
-          >
-          and
-          <a href="https://github.com/esphome/bluetooth-proxies/"
-            >configuration is on GitHub</a
-          >.
-        </i>
-      </p>
+      <div>
```
Comment (balloob) [inline static/index.html:244]: I don’t think the div adds to rendering here

File: static/index.html (hunk 1 above)
Comment (bdraco) [reply static/index.html:211]: Yes, otherwise its a 404

File: static/index.html (hunk 2 above)
Comment (bdraco) [reply static/index.html]: ba8407d8a1e6701f14d2b7ac3eaa6512570059b0

File: static/index.html (hunk 1 above)
Comment (balloob) [reply static/index.html:211]: Noticed this:

CleanShot 2022-09-06 at 21 04 45

**Pull request merged**